commands/config_test.go

```diff
@@ -1,104 +0,0 @@
-package commands
```

Comment:
Will new unit tests for the new code appear at some point? If so, it might be useful to keep this empty (mostly) file around as a reminder that tests need to be written...